src/FlatButtonConfirm.js

```diff
@@ -45,7 +45,6 @@ export default class PopoverExampleSimple extends React.Component {
             confirmLabel,
             content,
             style,
-            disabled,
```

Comment:
+1 for tests. I should have passed this prop to the control button. We can leave, I'll catch it again when creating a guide example for this. It's a wip component.